.devcontainer/devcontainer.json

```diff
@@ -2,35 +2,37 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.194.3/containers/dapr-javascript-node
 {
 	"name": "Vehicle Model Generator",
-	"build": {
-		"dockerfile": "Dockerfile"
-	},
-	"runArgs": [
-		"--init",
-		"--privileged"
-	],
-	"settings": {
-		"python.pythonPath": "/usr/bin/python3",
-		"python.defaultInterpreterPath": "/usr/bin/python3",
-		"python.linting.enabled": true,
-		"python.formatting.provider": "black",
-		"python.linting.banditEnabled": true,
-		"python.linting.flake8Enabled": true,
-		"python.disableInstallationCheck": true,
-		"terminal.integrated.defaultProfile.linux": "zsh",
-		"terminal.integrated.profiles.linux": {
-			"zsh": {
-				"path": "/usr/bin/zsh"
-			}
+    "image": "ghcr.io/eclipse-velocitas/devcontainer-base-images/python:v0.2",
```

Comment:
either remove the dockerfile completely or use it here

Reply:
Done.